regression/README.md

```diff
@@ -1,10 +1,10 @@
 Here is an example to run a given regression suite: 
 
-```
-python3 testing_tool.py --regression floats --tool /Users/lucascordeiro/esbmc/build/src/esbmc/esbmc --mode CORE
-```
```

Comment:
Do we need to remove this command line example? I think it is useful to know how to run the Python script manually. `ctest` is just another way to invoke the script in an automated fashion.

Reply:
The test discovery of the testing tool was removed. So it is not working anymore.

Reply:
I see, thanks.